tests/test_campaigns.py

```diff
@@ -62,19 +62,15 @@ def test_campaign_login_logged_out(self):
     def test_campaign_landing_logged_in(self):
         ensure_schemas()
         for key, value in self.campaigns.items():
-            resp = self.app.get(value['url_landing'], auth=self.user.auth)
+            resp = self.app.get(value['url_landing'], auth=self.user.auth).maybe_follow()
```

Comment:
Campaign url before: `https://osf.io/preprints/`
Campaign url now: `https://osf.io/login?next=https://osf.io/preprints/`
`maybe_follow()` handles the one extra redirection.